### ISSUE TITLE
Debugging assistance

### ISSUE THREAD

**Issue by toddinpal** [stjohnjohnson/smartthings-mqtt-bridge#58]
So are there any suggestions for debugging this bridge?  I have it set up as described running in a docker container on a Synology NAS.  As far as I can tell I don't have anything flowing through it yet.  Using mosquitto_sub I don't see any messages related to this bridge.  The events.log file has nothing interesting in it.  It would be nice if there were a simple way to enable some debugging logging.

-tl

**Comment by johnypony3** [comment]
I also would like to know

**Comment by stetho** [comment]
I also have it running on Docker on a Synology NAS and it doesn't seem to be doing anything.

Did you make any progress?

**Comment by johnypony3** [comment]
Although i dont have synology, i have it working on my plex. For me it started working, but then stopped when i decided to move from docker to a npm service. During debugging i found that the mosquitto log had socket errors from the bridge (same machine). I knew that the queue wasnt the problem because i have homebridge writing to that queue no problem.
I was at wits end, i ended up resetting the ST hub (what a pia), uninstalling and reisntalling this module. And it started working.
I can give you what hopefully is a hint, if this bridge is started manually, it will record every call. But only if its able write to the queue unfortunately. 
I can tell you that it is very stable and easily glues my homebridge and ST devices. If you have issues, post up configs for the bridge and any logs ill try to help

**Comment by stetho** [comment]
I just installed Docker on my Mac Mini and tried this - it worked first time. It looks like it might be a Synology problem.

**Comment by johnypony3** [comment]
can tell you, its not a synology problem. its just not happy sometimes. i am consolidating everything into docker, and its literally not working right now for me. i have already deleted and recreated the device with no luck.

edit (got it working):
what i noticed is that the web server was not starting by looking at the log. in my case, the port that it is listening on was taken by another docker container.

tl;dr; this thing works, it just doesnt give you any valid errors
